test/test_unary_ufuncs.py

```diff
@@ -419,6 +419,9 @@ def _test_out_arg(self, op, input, output, expected):
 
     @ops(unary_ufuncs, dtypes=OpDTypes.supported)
     def test_out_arg_all_dtypes(self, device, dtype, op):
+        if not op.supports_tensor_out:
```

Comment:
Won't be required if out variant is supported. All other UnaryOps have out= variant.